hoomd/pytest/test_snapshot.py

```diff
@@ -6,14 +6,6 @@
 import numpy
```

Comment:
This test file appears to have all skipif logic for `gsd` removed. Don't we want to importorskip in `test_gsd_from_snapshot_empty` and `test_from_gsd_snapshot_populated`?